plugins/chat/spec/system/navigating_to_message_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+RSpec.describe "Navigating to message", type: :system, js: true do
+  fab!(:current_user) { Fabricate(:user) }
+  fab!(:channel_1) { Fabricate(:category_channel) }
+  fab!(:first_message) { Fabricate(:chat_message, chat_channel: channel_1) }
+
+  let(:chat_page) { PageObjects::Pages::Chat.new }
+  let(:chat_drawer_page) { PageObjects::Pages::ChatDrawer.new }
```

Comment:
We should move this inside each context

Reply:
not sure I follow? You mean it should always be available?

Reply:
Nono, I mean each `let` is only used inside a single context block.

Reply:
OHHH ok, sorry I misunderstood